### PR TITLE
Muhammad

### PR DESCRIPTION
**Pull request by techbone**
I have completed my task on the SelfAffirmation page for the basic user, I am having slight issues understanding the UX though.

### CODE REVIEW

**Comment by thisdothassan** [comment]
what issue do you have with the ux

File: yarn.lock
Comment (thisdothassan) [inline yarn.lock:1]: Delete this, we are using npm as our package manager

File: src/pages/admin/SelfAffirmation.jsx [hunk 1]
```diff
@@ -1,76 +1,115 @@
-import React from 'react';
-import Button from '../../components/common.jsx/button';
+import React from "react";
+import Button from "../../components/common.jsx/button";
 
 const SelfAffirmation = () => {
-    return ( 
-        <div className='px-4 bg-white'>
```
Comment (thisdothassan) [inline src/pages/admin/SelfAffirmation.jsx:6]: why did you make changes to the admin selfAffirmation page ? If you want to lift things from there you can do that but don't make changes to what you're not working on

File: public/images/FlipCards.svg
Comment (thisdothassan) [inline public/images/FlipCards.svg:1]: I think there's a way to achieve this with CSS, I don't want us to overload our server with images when it's not necessary. Can you find a way around it?

**Comment by thisdothassan** [comment]
1. Are you the one using yarn as your package manager? if yes kindly delete your package-lock.json and yarn.lock and use npm to install dependencies again by running `npm i`
2. The way you name your card image is too generic, think about it we can have some other cards in the future. Kindly rename it to something customised for it's purpose (I guess it's self affirmation) .

make the necessary changes and raise a new PR

File: src/pages/admin/SelfAffirmation.jsx (hunk 1 above)
Comment (techbone) [reply src/pages/admin/SelfAffirmation.jsx:6]: It was when I was having issues trying to render the page the first time, but it's resolved now.

File: public/images/FlipCards.svg
Comment (techbone) [reply public/images/FlipCards.svg:1]: Oh i don't know of a way, I just use the svg the way I export it from the figma file whenever I have to.

**Pull request closed without merging**